events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -204,52 +197,48 @@ <h4>Choose Question Details (Optional) <br>& Click 'New Question'</h4>
-                        triviaData = JSON.parse(event);
-                        console.log(event);
-                        if(triviaData.application !== "trivia") {
-                            return;
-                        } 
-                        else if ("question" in triviaData) {
-                            document.getElementById("question").innerHTML = triviaData.question;
-                            if (triviaData.type === "multiple") {
-                                document.getElementById("answer").innerHTML =  "<p class=\"correct\">" + 
-                                    triviaData.correct_answer + "<p>" + 
-                                    triviaData.incorrect_answers[0] + "<p>" + 
-                                    triviaData.incorrect_answers[1] + "<p>" +
-                                    triviaData.incorrect_answers[2];
-                            } else {
-                                document.getElementById("answer").innerHTML = triviaData.correct_answer;
-                            }
-                        } 
-                        else if ("response_code" in triviaData) {
-                            document.getElementById("question").innerHTML = "Response: " + triviaData["response_code"] + " Check with your trivia database or HiFi Admin, something is clearly wrong";
-                            databaseButton.disabled = true;
-                            typeButton.disabled = true;
-                            diffButton.disabled = true;
-                            catButton.disabled = true;
-                            beginButton.disabled = false;
-                            endButton.disabled = true;
-                            newQButton.disabled = true;
-                            showQButton.disabled = true;
-                            answerButton.disabled = true;
-                        } 
-                        else {
-                            document.getElementById("question").innerHTML = "You entered an invalid URL, tab name, or have not properly setup your trivia database for access by this application. Try using the 'Default Catalog' instead.";
-                            document.getElementById('catalog').value = "Default Catalog" + "   \u25BC";
-                            databaseButton.disabled = true;
-                            typeButton.disabled = true;
-                            diffButton.disabled = true;
-                            catButton.disabled = true;
-                            beginButton.disabled = false;
-                            endButton.disabled = true;
-                            newQButton.disabled = true;
-                            showQButton.disabled = true;
-                            answerButton.disabled = true;
-                        }       
-                    } catch (e){
-                        console.log(e, "error");
-                    }
+                    triviaData = JSON.parse(event);
```

Comment:
this is still necessary to put in a `try` block, because you may not always be able to parse `event`